docs/USAGE_ADVANCED.md

```diff
@@ -390,9 +400,16 @@ remote_theme: eecs485staff/primer-spec
 primerSpec:
   sitemap:
     label: My custom sitemap label
+    externalLinks:
+      - title: Title1
+        url: URL of External Page
+      - title: Title2
+        url: URL of Second External Page
   # ... (other site configuration options)
 ```
 
+In this example, all internal pages will appear first in an order based on the individual pages' [`sitemapOrder`](#sitemaporder-number) property. After them will be Title2 followed by Title1.
```

Comment:
Wait, shouldn't it be Title1 then Title2?

Reply:
You're right--this is something that changed after I changed the API. I'll fix it!